### ISSUE TITLE
Random status page name

### ISSUE THREAD

**Issue by Lanhild** [louislam/uptime-kuma#5420]
### 📑 I have found these related issues/pull requests

n/a

### 🏷️ Feature Request Type

Status-page

### 🔖 Feature description

We should be able to generate a random string or UUID for status page names

### ✔️ Solution

The same behavior as a UUID or something like random.org;
![image](https://github.com/user-attachments/assets/15b7be84-b4a7-4dff-b339-f4a25838f7a8)

**Comment by circlecrystalin** [comment]
@CommanderStorm How do you think about this feature? I think, it's worth for our uptime-kuma.
If you think so, I will create pr for this feature.

**Comment by CommanderStorm** [comment]
I think usually, status pages are for humans.
We also consider everything on status pages to be public and "expose" things such as badges only if they are on there.

I think by random status page blob generation, we would provide a false sense of security.
I would like for the "we want to publish this, but actually not" crowd to put this behind an reverse proxy instead.
A "fill this with randomness" does not solve a problem I think.

@Lanhild Did I miss your usecase?

**Comment by CommanderStorm** [comment]
thanks for the ping @circlecrystalin

**Comment by Lanhild** [comment]
> I think usually, status pages are for humans. We also consider everything on status pages to be public and "expose" things such as badges only if they are on there.
> 
> I think by random status page blob generation, we would provide a false sense of security. I would like for the "we want to publish this, but actually not" crowd to put this behind an reverse proxy instead. A "fill this with randomness" does not solve a problem I think.
> 
> [@Lanhild](https://github.com/Lanhild) Did I miss your usecase?

Yes, you did. My main usecase for this was for customers' status pages. When hosting apps as a business, it's often innapropriate to have the same status page for all customers, especially if different services are involved. 

Being able to generate a status page slug is simply a QoL feature, to save time when having to create X number of status pages

**Comment by CommanderStorm** [comment]
But how do you then know which customer this belongs to afterwards?
I am not sure if this would actually make your live simpler.
